sdk/core/core-http/src/webResource.ts

```diff
@@ -328,7 +328,7 @@ export class WebResource implements WebResourceLike {
         baseUrl +
         (baseUrl.endsWith("/") ? "" : "/") +
         (pathTemplate.startsWith("/") ? pathTemplate.slice(1) : pathTemplate);
-      const segments = url.match(/({[\w\-]*\s*[\w\-]*})/gi);
+      const segments = url.match(/({[\w-]*\s*[\w-]*})/gi);
```

Comment:
did prettier do this? It's technically correct and identical, but seems slightly less obvious

Reply:
Not prettier, this was eslint. We have an eslint rule that doesn't like unnecessary escaping.

Reply:
huh, okay.